### PR TITLE
Expose layer_norm variance_epsilon as an argument

### PR DESCRIPTION
**Pull request by KyotoSunshine**
In some cases it is necessary to change the epsilon value in order to avoid numerical instabilities during gradient computation. 
This makes it consistent with batch_norm, which also exposes the epsilon value. The epsilon is exposed with the same name (i.e. "epsilon") as in batch norm.

### CODE REVIEW

**Comment by martinwicke** [comment]
contrib is deprecated and will soon disappear. The core layer has an epsilon parameter already. I will close this PR.
